libraries/redcore/api/hal/hal.php

```diff
@@ -2083,6 +2158,36 @@ public function triggerFunction($functionName)
 	}
 
 	/**
+	 * Calls method from defined object as some Joomla methods require referenced parameters
+	 *
+	 * @param   object  $object        Object to run function on
+	 * @param   string  $functionName  Function name
+	 * @param   array   $args          Arguments for the function
+	 *
+	 * @return mixed Result from callback function
+	 */
+	public function triggerCallFunction($object, $functionName, $args)
```

Comment:
This function is avoiding passing the data as an array when arguments are too few, right?